### ISSUE TITLE
TypeError: declutterTree.collides is not a function

### ISSUE THREAD

**Issue by conbrad** [openlayers/openlayers#15854]
**Describe the bug**
Map doesn't load and throws `TypeError: declutterTree.collides is not a function`

**To Reproduce**
1. Install `ol` version 9.2.2
2. Load an openlayers map with a vector tile layer, presumably with `declutter` set to `true` and a `olpmtiles.PMTilesVectorSource`
3. See error: 

```
TypeError: declutterTree.collides is not a function
    at Executor.execute_ (Executor.js:959:1)
    at Executor.execute (Executor.js:1263:1)
    at ExecutorGroup.execute (ExecutorGroup.js:401:1)
    at CanvasVectorTileLayerRenderer.postRender (VectorTileLayer.js:775:1)
    at CanvasVectorTileLayerRenderer.renderFrame (TileLayer.js:535:1)
    at VectorTileLayer.render (Layer.js:372:1)
    at CompositeMapRenderer.renderFrame (Composite.js:131:1)
    at Map.renderFrame_ (Map.js:1570:1)
    at Map.animationDelay_ (Map.js:1424:1)
```

**Expected behavior**
Map should load without error

**Comment by ahocevar** [comment]
Duplicate of #15849, fixed with #15851.

**Comment by tschaub** [comment]
This is resolved in the `ol@9.2.3` release (thanks @ahocevar for the fix)
